main.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'app'
+app = App.new
+
+def choices(app)
+  puts 'Welcome to School Library App!'
+  loop do
+    puts 'Please choose an option by enterin a number:'
+    puts '1 - List all books'
```

Comment:
- [ ] When I'm trying to create a rental, the list of people is not listed, an error is formed and I'm not able to create a rental.

https://www.loom.com/share/72189f7fa61144b1bde0ed2824f9b498?sid=899137d2-5511-4871-b600-1928e61158dd